OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix NewProfile link

# Description
Processed a meaningless folde. Updated lint ignore file .



## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. check all function is normal
